src/mapml/handlers/ContextMenu.js

```diff
@@ -17,17 +17,21 @@ export var ContextMenu = L.Handler.extend({
     //setting the items in the context menu and their callback functions
     this._items = [
       {
-        text: M.options.locale.cmBack + " (<kbd>B</kbd>)",
+        text: M.options.locale.cmBack + " (<kbd>Alt+Left Arrow</kbd>)",
         callback:this._goBack
       },
       {
-        text: M.options.locale.cmForward + " (<kbd>F</kbd>)",
+        text: M.options.locale.cmForward + " (<kbd>Alt+Right Arrow</kbd>)",
```

Comment:
Looks good, just dont forget to add localized versions of this in the extension, similar to what you did for "Exit Fullscreen". Create a PR for the extension.

Nice Work on this!